### PR TITLE
1.22 finilized major themes

### PR DESCRIPTION
**Pull request by pmmalinov01**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:
To complete release notes for Kubernetes 1.22
#### Special notes for your reviewer:
/cc @sladyn98 @cici3§ @Damans227 @simcard0000 @savitharaghunathan

### CODE REVIEW

File: releases/release-1.22/release-notes/release-notes-draft.md [hunk 1]
```diff
@@ -1,3 +1,81 @@
...
+
+### Deploy a more secure control plane with kubeadm
+
+A new alpha feature allows running the kubeadm control plane components as non-root users. This is a long requested security measure in kubeadm. To try it you must enable the kubeadm-specific `RootlessControlPlane` feature gate. When you deploy a cluster using this alpha feature, your control plane runs with lower privileges.
+
+A new [v1beta3 configuration API](https://github.com/kubernetes/kubeadm/issues/1796). It [iterates over v1beta2](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3) by adding some long requested features and deprecating some existing ones. The `V1beta3` is now the preferred API version; the `v1beta2` API also remains available and is not yet deprecated.
+
+### etcd moves to version 3.5.0
+
+Kubernetes' default backend storage, etcd, has a new release 3.5.0 and the community embraced it. The new release comes with improvements to the Security, performance, monitoring and developer experience. There are numremains available and is not yet deprecated.
```
Comment (Damans227) [inline releases/release-1.22/release-notes/release-notes-draft.md:44]: "_There are numremains available and is not yet deprecated_."

What is numremains ? If it's some sort of a command, should we enclose it in back ticks ?

File: releases/release-1.22/release-notes/release-notes-draft.md [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+
+### etcd moves to version 3.5.0
+
+Kubernetes' default backend storage, etcd, has a new release 3.5.0 and the community embraced it. The new release comes with improvements to the Security, performance, monitoring and developer experience. There are numremains available and is not yet deprecated.
```
Comment (Damans227) [inline releases/release-1.22/release-notes/release-notes-draft.md:48]: Same doubt as above.

File: releases/release-1.22/release-notes/release-notes-draft.md [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+
+### etcd moves to version 3.5.0
+
+Kubernetes' default backend storage, etcd, has a new release 3.5.0 and the community embraced it. The new release comes with improvements to the Security, performance, monitoring and developer experience. There are numremains available and is not yet deprecated.
```
Comment (Damans227) [inline releases/release-1.22/release-notes/release-notes-draft.md:52]: This one too.

File: releases/release-1.22/release-notes/release-notes-draft.md [hunk 4]
```diff
@@ -1,3 +1,81 @@
...
+
+### etcd moves to version 3.5.0
+
+Kubernetes' default backend storage, etcd, has a newhange lets administrators opt in to configuring swap on Linux nodes, treating a portion of block storage as additional virtual memory.
```
Comment (Damans227) [inline releases/release-1.22/release-notes/release-notes-draft.md:56]:
```suggestion
Kubernetes' default backend storage, etcd, has a new change which lets administrators opt in to configuring swap on Linux nodes, treating a portion of block storage as additional virtual memory.
```

File: releases/release-1.22/release-notes/release-notes-draft.md [hunk 5]
```diff
@@ -1,3 +1,81 @@
...
+
+### Cluster-wide seccomp defaults
+
+A new alpha feature gate SeccompDefault has been added to the kubelet, together with a corresponding command line flag `--seccomp-default` and kubelet configuration. If both are enabled, then the kubelet's behavior changes for pods that don't explicitly set a seccomp profile.
```
Comment (Damans227) [inline releases/release-1.22/release-notes/release-notes-draft.md:60]:
```suggestion
A new alpha feature gate `SeccompDefault` has been added to the kubelet, together with a corresponding command line flag `--seccomp-default` and kubelet configuration. If both are enabled, then the kubelet's behaviour changes for pods that don't explicitly set a seccomp profile.
```

**Review by Damans227 — CHANGES_REQUESTED** [review]

File: releases/release-1.22/release-notes/release-notes-draft.md (hunk 5 above)
Comment (pmmalinov01) [reply releases/release-1.22/release-notes/release-notes-draft.md:52]: Behavior can stay as the American variant. We are not that strict.

File: releases/release-1.22/release-notes/release-notes-draft.md (hunk 1 above)
Comment (pmmalinov01) [reply releases/release-1.22/release-notes/release-notes-draft.md]: This was due to copy/paste and bad interaction with vim

**Review by Damans227 — APPROVED** [review]

**Pull request merged**